components/User/UserProfile/PrivatePublicUserProfile/UserDecks.js

```diff
@@ -1,48 +1,116 @@
 import React from 'react';
 import PopularDecks from '../PopularDecks';
+import { navigateAction } from 'fluxible-router';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { fetchUserDecks } from '../../../../actions/user/userprofile/fetchUserDecks';
+import { fetchNextUserDecks } from '../../../../actions/user/userprofile/fetchNextUserDecks';
 
 class UserDecks extends React.Component {
     constructor(props){
         super(props);
-        this.sortBy = '2';
+        this.messages = this.getIntlMessages();
     }
     componentDidMount() {
         $(this.refs.sortDropdown).dropdown({onChange: this.dropdownSelect.bind(this)});
     }
 
-    componentDidUpdate() {}
+    componentDidUpdate() {
+        $(this.refs.sortDropdown).dropdown({onChange: this.dropdownSelect.bind(this)});
```

Comment:
Why do you bind dropdownSelect on every update?

Reply:
The dropdown wasn't working correctly when not binding it on every update, but I solved that issue and removed that from the componentDidUpdate(). Thanks